gamification-evm-services/src/main/java/io/meeds/evm/gamification/model/ERC20Token.java

```diff
@@ -11,9 +11,9 @@
 @AllArgsConstructor
 public class ERC20Token {
```

Comment:
Shouldn't we consider a generic Contract DTO ?

```suggestion
public class EvmContract {
```